acceptance-tests/src/test/scala/ch/renku/acceptancetests/pages/GitLabPages.scala

```diff
@@ -92,4 +104,51 @@ class GitLabPages(
       }
     }
   }
+
+  case object ProjectBranchesPage extends GitLabPage {
+
+    override val path: Path = Refined.unsafeApply(
+      s"/gitlab/${userCredentials.userNamespace}/${projectDetails.title.toPathSegment}/-/branches"
+    )
+
+    override val title: Title = Refined.unsafeApply(
+      s"General · Branches · ${userCredentials.fullName} / ${projectDetails.title} · GitLab"
+    )
+
+    override def pageReadyElement(implicit webDriver: WebDriver): Option[WebElement] = Some(newBranchButton)
+
+    def newBranchButton(implicit webDriver: WebDriver): WebElement = eventually {
+      find(cssSelector(s"a[href='$path/new']")) getOrElse fail("New Branch button not found")
+    }
+  }
+
+  case object NewBranchPage extends GitLabPage {
```

Comment:
I don't see this object to be used anywhere.

Reply:
_Removed_.